Makefile

```diff
@@ -264,28 +264,49 @@ PLUGIN_IMAGE_NAME ?= docker-volume-rclone
 PLUGIN_IMAGE ?= $(PLUGIN_IMAGE_USER)/$(PLUGIN_IMAGE_NAME):$(PLUGIN_IMAGE_TAG)
 
 PLUGIN_BASE_IMAGE := rclone/rclone:latest
-PLUGIN_BUILD_DIR := ./build/docker-plugin
+PLUGIN_BUILD_DIR := ./build
 PLUGIN_CONTRIB_DIR := ./cmd/serve/docker/contrib/plugin
 PLUGIN_CONFIG := $(PLUGIN_CONTRIB_DIR)/config.json
 PLUGIN_DOCKERFILE := $(PLUGIN_CONTRIB_DIR)/Dockerfile
 PLUGIN_CONTAINER := docker-volume-rclone-dev-$(shell date +'%Y%m%d-%H%M%S')
 
-docker-plugin: docker-plugin-rootfs docker-plugin-create
+PLUGIN_PLATFORM ?= linux/amd64,linux/386,linux/arm64,linux/arm/v7
 
-docker-plugin-image: rclone
-	docker build --no-cache --pull --build-arg BASE_IMAGE=${PLUGIN_BASE_IMAGE} -t ${PLUGIN_IMAGE} -f ${PLUGIN_DOCKERFILE} .
+docker-plugin: docker-plugin-create docker-plugin-push
 
-docker-plugin-rootfs: docker-plugin-image
-	mkdir -p ${PLUGIN_BUILD_DIR}/rootfs
-	docker create --name ${PLUGIN_CONTAINER} ${PLUGIN_IMAGE}
-	docker export ${PLUGIN_CONTAINER} | tar -x -C ${PLUGIN_BUILD_DIR}/rootfs
-	docker rm -vf ${PLUGIN_CONTAINER}
-	cp ${PLUGIN_CONFIG} ${PLUGIN_BUILD_DIR}/config.json
+docker-plugin-build:
+	rm -rf ${PLUGIN_BUILD_DIR}
 
-docker-plugin-create:
-	docker plugin rm -f ${PLUGIN_IMAGE} 2>/dev/null || true
-	docker plugin create ${PLUGIN_IMAGE} ${PLUGIN_BUILD_DIR}
+	docker buildx create --use \
+		--platform ${PLUGIN_PLATFORM} \
+		--name ${PLUGIN_CONTAINER}
+
+	docker buildx build --no-cache --pull \
```

Comment:
Another good side of `buildx` is its ability to build multiple outputs (or parts of multi-arch image) in parallel greatly reducing build time. We would not gain a lot for docker plugin from concurrent build because now it simply adds a single binary to the `apline` image and adjusts a few settings. However the rclone release build would gain a lot. That step is based on an external buildx-based package [ilteoood/docker_buildx](https://github.com/ilteoood/docker_buildx/releases) so I think we should monitor for new versions and upgrade when it starts to support concurrency.